db/schema.rb

```diff
@@ -32,14 +32,16 @@
     t.boolean "has_dbs_check"
     t.date "dbs_achieved_on"
     t.boolean "has_first_aid_training"
-    t.date "first_aid_achieved_on"
+    t.date "first_aid_expires_at"
```

Comment:
This looks like it's not meant to be here :)

Reply:
Yeah weird things can happen when you're working with DB migrations in different branches! What happens when you run a new migration is that it auto-generates the `schema.db` based on what's in your local database, so if you were working on a different migration on another branch before and you didn't roll back the changes before switching to the new branch, then whatever other database changes you were working on will end up in your new branch's schema.

Reply:
Maybe try rolling back to where you don't have the new SENCO early years fields in your DB (unless they are already not there because I don't see them in the schema anymore), then merging in main and running migrations?

Reply:
I think it's fixed now :)